tests/cli_test.py

```diff
@@ -33,3 +34,24 @@ def test_cli_posargs(capsys, monkeypatch):
     stdout, stderr = capsys.readouterr()
     assert not stderr
     assert stdout == " example com\n bbc co.uk\nforums bbc co.uk\n"
+
+
+def test_cli_namedargs(capsys, monkeypatch):
```

Comment:
I appreciate the increase in test coverage. I'm going to cherry-pick it outside of this PR, because it distracts from the main theme of the PR, the IP bugfix.